### DISCUSSION TITLE
Allow hearting/liking shared photos

### DISCUSSION THREAD

**Discussion by UnderTheCarpet** [ente-io/ente#792] [Enhancements]
**Describe the feature**
As a collaborator of a shared album I should be able to heart/like photos from that album.

If two persons share an album and both upload photos to it, it is weird if the UX depends on the uploader.

**Comment by vishnukvmd** [comment]
Hey @clamydo, thanks for the feedback, we'll add this in one of our future releases.

**Comment by vishnukvmd** [comment]
Related https://github.com/ente-io/ente/discussions/790
